### ISSUE TITLE
[Content item horizontal]: QA: Screen reader does not identify the Thumbnail image

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9657]
### Description

Page Element:
With Thumbnail

Issue Description:
Screen reader does not identify the Thumbnail image

Recommend Fix:
If these Icons are informative, Provide alt text that describes the purpose. if these are decorative, hide these from the screen reader using alt="".

User Impact:
When information is conveyed only through icons, images or graphics and has no appropriate text alternative, screen reader users are unable to understand it.

Tool:
Voice Over_iPhone
JAWS_Desktop

### Component(s) impacted

Content item horizontal

### Browser

Safari, Firefox

### Carbon for IBM.com version

v1.40.0-rc.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components/?path=/story/

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
